lib/schema-loader.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const vm = require('vm');
+const { all: decorators } = require('./decorators');
+
+const SCRIPT_TIMEOUT = 2000;
+
+// Process schema source
+//   name - <string> schema name
+//   source - <string> schema source
+//   context - <Object> object to be assigned to global during loading
+// Returns: <Object> processed schema
+const processSchema = (
+  name, source, context = {}, evaluator = vm.runInNewContext
```

Comment:
Is there a need to make `evaluator` configurable?

Reply:
In order to allow to use this without polyfill in browser.
/cc @aqrln

Reply:
Oh, OK then, I just thought we were going to use `webpack` to polyfill `vm` for browsers.